client/js/jsonedit.js

```diff
@@ -964,6 +978,11 @@ function jeShowCommands() {
   }
   delete activeCommands[""];
 
+  if(jeContext[jeContext.length-1] == '(var expression)') {
+    commandText += `\n  <b>var expression</b>\n<label>Search </label><input id="var_search" name="var_search" type="text" value ="${jeKeyword}"><br>`;
```

Comment:
This breaks if the user enters `"`, right? Maybe just do `(...).value = ...` after line 1022.

Reply:
Doesn't break for me. I don't understand your suggestion about value, though.

Reply:
Enter `"><b>test<` into the search box and then click into another expression in the editor (so it rebuilds the command pane). Because the `"` is not escaped, it looks like this:

![image](https://user-images.githubusercontent.com/73538315/120312689-55511100-c2d9-11eb-93af-66103b3cc05f.png)

Instead, set the value like this: `$('#var_search').value = jeKeyword;` after the HTML was added to the command pane. Setting the value takes care of escaping HTML special characters `"`, `<` and `>`.